### ISSUE TITLE
Use the lockfile in `rfbrowser init`

### ISSUE THREAD

**Issue by xylix** [MarketSquare/robotframework-browser#159]
An issue caused by us not having an npm lockfile can bve seen here: https://github.com/MarketSquare/robotframework-browser/pull/158/checks?check_run_id=906692387

Using `yarn install` in the CI isn't really a solution, since we do probably want to support users without yarn installed. Possible solutions would be to use yarn to produce an npm shrinkwrap file or to move to npm.

**Comment by mkorpela** [comment]
I think we should just update this to latest https://github.com/MarketSquare/robotframework-browser/blob/fbd0dae79cc8a2bfa285f503b254d46b1b4c4553/package.json#L35

**Comment by mkorpela** [comment]
And update tests accordingly. We are floating on top of a library that is under constant development.

**Comment by mkorpela** [comment]
I've added CI run ones a day to ensure that we are compatible with the latest dependency versions https://github.com/MarketSquare/robotframework-browser/commit/0c9b838467620ecc28ef8fffb065d5c22c09ed7d

I've also updated playwright dependency minimum requirement to the latest 1.2.1 version.

Currently I think this resolves the issue.

We can float on top of most recent Playwright until there is a really good reason to lock it to a specific older version.
